app/api/contests/set_additional_time.go

```diff
@@ -23,8 +23,9 @@ import (
 //
 //                Restrictions:
 //                  * `item_id` should be a timed contest;
-//                  * the authenticated user should have `bCachedAccessSolutions` or `bCachedFullAccess` on the input item;
-//                  * the authenticated user should own the `group_id`.
+//                  * the authenticated user should have `solutions` or `full` access on the input item;
+//                  * the authenticated user should own the `group_id`;
+//                  * if the contest is timed (`items.bHasAttempts` = 1), then the group should not be a user group.
```

Comment:
"if the contest is timed (`items.bHasAttempts` = 1)" -> " contest is timed" is not related with `items.bHasAttempts`, mistake?

Reply:
fixed